docs/proposals/old-timer.md

```diff
@@ -0,0 +1,146 @@
+# Heapster Oldtimer
+
+## Overview
+
+Prior to the Heapster refactor, the Heapster model presented aggregations of
+metrics over certain time periods (the last hour and day).  Post-refactor, the
+concern of presenting an interface for historical metrics was to be split into
+a separate Heapster component: Oldtimer.
```

Comment:
I would suggest to include it in the default deployment. I think that a for 99% deployments a single instance will be enough. It should rather not consume lots of memory. 
I think that anyone who needs to perform large-scale data analysis will do it in some other way (not via this "poor" api).